src/ppl/test/add_constraint12.tcl

```diff
@@ -0,0 +1,17 @@
+# gcd_nangate45 IO placement
+source "helpers.tcl"
+read_lef Nangate45/Nangate45.lef
+read_def gcd.def
+
+set_io_pin_constraint -pin_names {resp_msg[3] resp_msg[2] resp_msg[14] req_val} -region top:0-18
```

Comment:
Does this cover the partially mirrored group case?

Reply:
I've added a new test covering it.